data/wanderer/wanderers.txt

```diff
@@ -696,6 +696,65 @@ phrase "hostile disabled wanderer"
 	word
 		"."
 
+event "wanderers: all facilities available"
```

Comment:
I think that instead of having three events separated in time that copy/paste many of the same planets, you should have three events separated by which planets they affect. All three events would be triggered for the first compatibility mission, two events for the second, and only one event for the third, removing the event that updates the captured planets at the respective compatibility mission that excludes them now.